pkg/sql/pgwire/testdata/pgtest/portals

```diff
@@ -0,0 +1,29 @@
+# Verify that a completed portal can't be re-executed.
+
+send
+Parse {"Query": "SELECT 1"}
+Bind
+Execute
+Sync
+----
+
+until
+ReadyForQuery
+----
+{"Type":"ParseComplete"}
+{"Type":"BindComplete"}
+{"Type":"DataRow","Values":[{"text":"1"}]}
+{"Type":"CommandComplete","CommandTag":"SELECT 1"}
+{"Type":"ReadyForQuery","TxStatus":"I"}
+
+send
+Execute
+Sync
+----
+
+until
+ErrorResponse
+ReadyForQuery
+----
+{"Type":"ErrorResponse"}
```

Comment:
it'd be cool if this could include the error message. It'd make this test a lot more readable.